$OpenBSD: patch-configure,v 1.5 2014/06/07 20:29:32 bcallah Exp $

1. Disable detection of pangoft2.

2. Disable detection of cppunit.  (--disable-cppunit is a NOP.)

3. Disable detection of ftgl. (no --disable-ftgl option.)

--- configure.orig	Fri May 16 15:15:32 2014
+++ configure	Sat Jun  7 14:03:02 2014
@@ -17495,8 +17495,7 @@ if test -n "$FTGL_CFLAGS"; then
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_FTGL_CFLAGS=`$PKG_CONFIG --cflags "ftgl >= 2.1.3" 2>/dev/null`
-		      test "x$?" != "x0" && pkg_failed=yes
+  pkg_failed=untried
 else
   pkg_failed=yes
 fi
@@ -17512,8 +17511,7 @@ if test -n "$FTGL_LIBS"; then
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_FTGL_LIBS=`$PKG_CONFIG --libs "ftgl >= 2.1.3" 2>/dev/null`
-		      test "x$?" != "x0" && pkg_failed=yes
+  pkg_failed=untried
 else
   pkg_failed=yes
 fi
@@ -18672,7 +18670,7 @@ fi
 
 # Build development tools?
 
-pkg_failed=no
+pkg_failed=yes
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for PANGOFT2" >&5
 $as_echo_n "checking for PANGOFT2... " >&6; }
 
@@ -18963,7 +18961,7 @@ fi
 
 # Use cppunit for unit tests?
 
-pkg_failed=no
+pkg_failed=yes
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for CPPUNIT" >&5
 $as_echo_n "checking for CPPUNIT... " >&6; }
 
